api/src/main/java/net/countercraft/movecraft/TrackedLocation.java

```diff
@@ -5,43 +5,73 @@
 import org.jetbrains.annotations.NotNull;
 
 public class TrackedLocation {
-    private MovecraftLocation offSet;
-    private final Craft craft;
+
+    private int x;
+    private int y;
+    private int z;
```

Comment:
Tracked locations are an offset from a craft's core.  Translating them every translation is unneeded overhead.

Reply:
See above